libs/orderbook/src/lib/orderbook-row.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import {
+  PriceCell,
+  Vol,
+  CumulativeVol,
+  formatNumber,
+} from '@vegaprotocol/react-helpers';
+
+interface OrderbookRowProps {
+  bidVol: number;
+  relativeBidVol?: number;
+  price: number;
+  askVol: number;
+  relativeAskVol?: number;
+  cumulativeRelativeAskVol?: number;
+  cumulativeRelativeBidVol?: number;
+  decimalPlaces: number;
+}
+
+export const OrderbookRow = React.memo(
+  ({
+    bidVol,
+    relativeBidVol,
+    price,
+    askVol,
+    relativeAskVol,
+    decimalPlaces,
+    cumulativeRelativeAskVol,
+    cumulativeRelativeBidVol,
+  }: OrderbookRowProps) => {
+    return (
+      <>
```

Comment:
I'm using display: grid, I will need to switch to 
```
<div className="flex" role="row">
  <div className="grow-0 shrink-0 w-1/4"  role="gridcell"></div>
  <div className="grow-0 shrink-0 w-1/4" role="gridcell"></div>
  <div className="grow-0 shrink-0 w-1/4" role="gridcell"></div>
  <div className="grow-0 shrink-0 w-1/4" role="gridcell"></div>
</div>
```
instead of 
```
<>
  <div></div>
  <div></div>
  <div></div>
  <div></div>
</>
```